dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -118,6 +118,24 @@ namespace Microsoft.Windows.AppNotifications.Builder
         AppNotificationComboBox SetSelectedItem(String id);
     };
 
+    [contract(AppNotificationBuilderContract, 2)]
+    runtimeclass AppNotificationComboBox2
+    {
+        AppNotificationComboBox2(String id);
+
+        String Title;
+        String SelectedItem;
+
```

Comment:
Not a fan of exposing the class innards this way. Having a getter seems resonable but a setter seems more trouble than it is worh. These two setters, don't offer any validation contrairy to the fluent setter which do some validation.

Even worse, the fluent setter for the title encodes the value into xml, while the basic setter for the title does not. If the caller doesn't properly encode the value, then the builder, at best will produce a notification that can't be used, at worse, it will crash! This pattern is in use throught all the builder API and addressing it is beyond the scope of this PR.

Reply:
Exposing these setters is moreso for C# developers. https://github.com/microsoft/WindowsAppSDK/pull/2648#discussion_r924785223

Reply:
Then shouldn't we apply the same level of validation we apply to fluent setters? We shouldn't let external developers just set these values without doing any validation on them.

Reply:
Fair enough. Innards a spilling out once more...